f/frizzle/alerts/alerts_gcs.py

```diff
@@ -379,106 +379,67 @@ def _table_exists(self, cursor, table_name):
-                    logger.error(
-                        f"An error occurred while creating new column: {column} for {table_name}__metadata: {e}"
-                    )
-                    raise
-
-    def _create_alert_cols(self, table_name):
-        # TODO: have this reflect the create_table.sql file
-        alert_cols = [
-            "_id",
-            "alert_source",
-            "alert_type",
-            "area_alert_ha",
-            "basin_id",
-            "count",
-            "date_end_t0",
-            "date_end_t1",
-            "date_start_t0",
-            "date_start_t1",
-            "grid",
-            "label",
-            "length_alert_km",
-            "month_detec",
-            "sat_detect_prefix",
-            "sat_viz_prefix",
-            "satellite",
-            "territory_id",
-            "territory_name",
-            "year_detec",
-            "g__coordinates",
-            "g__type",
-            "geom",
-            "source",
-        ]
+                logger.info(f"Table {metadata_table_name} created.")
 
+    def _create_alerts_table(self, table_name):
         with self._get_conn() as conn, conn.cursor() as cursor:
-            if not self._table_exists(cursor, f"{table_name}__metadata"):
+            if not self._table_exists(cursor, table_name):
                 query = f"""
-                CREATE TABLE public.{table_name}();
+                CREATE TABLE {table_name}
+                (
+                    _id character varying(36) NOT NULL,
+                    -- These are found in "properties" of an alert Feature:
+                    alert_type text,
+                    area_alert_ha double precision,
+                    basin_id bigint,
+                    count bigint,
+                    date_end_t0 text,
+                    date_end_t1 text,
+                    date_start_t0 text,
+                    date_start_t1 text,
+                    grid bigint,
+                    label bigint,
+                    month_detec text,
+                    sat_detect_prefix text,
+                    sat_viz_prefix text,
+                    satellite text,
+                    territory_id bigint,
+                    territory_name text,
+                    year_detec text,
+                    length_alert_km double precision,  -- only present for linestring
+                    -- Deconstruct the "geometry" of a Feature:
+                    g__type text,
+                    g__coordinates text,
+                    -- Added by us
+                    source text,
+                    alert_source text
+                );
                 """
                 cursor.execute(query)
                 conn.commit()
                 logger.info(f"Table {table_name} created.")
-            else:
-                return
-            for column in alert_cols:
-                try:
-                    cursor.execute(f"""
-                    ALTER TABLE {table_name}
```

Comment:
An implication of removing this is that we no longer persist never-before-seen columns that may appear in the alerts.

If/when those show up, we insert the rest of the row but throw away the new field.